docs/getting-started/github-apps/github-apps-troubleshooting-tips.rst

```diff
@@ -54,6 +54,25 @@ you'll need to push another commit after correcting it to activate the sync to D
     - Are all indentation levels correct?
     - Does the name field match, if applicable?
 
+The changes made to my GitHub repo aren't appearing on Dockstore, but I've already installed the GitHub app and made the ``.dockstore.yml`` file. How can I figure out what's going wrong?
+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+If you've already tried waiting a couple of minutes and refreshing the browser on the My Workflows page, you can view GitHub App logs through Dockstore to see if there have been any errors.
```

Comment:
Best to safe I guess, especially if we grow a lot and start slowing down due to a massive number of requests, but my experience is we can know in seconds whether it worked or not.